sdk/identity/azure-identity/azure/identity/_credentials/certificate.py

```diff
@@ -29,8 +29,8 @@ class CertificateCredential(ClientCredentialBase):
     :keyword password: The certificate's password. If a unicode string, it will be encoded as UTF-8. If the certificate
           requires a different encoding, pass appropriately encoded bytes instead.
     :paramtype password: str or bytes
-    :keyword bool send_certificate: if True, the credential will send public certificate material with token requests.
-          This is required to use Subject Name/Issuer (SNI) authentication. Defaults to False.
+    :keyword bool send_certificate_chain: if True, the credential will send public certificate material with token
```

Comment:
since changing the name perhaps you want to update the comment to indicate the credential will send the full x5c certificate chain. Perhaps, since we're omitting x5c from the name it should at least be in the doc comments for this parameter?